css/styles.css

```diff
@@ -0,0 +1,176 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  width: 800px;
+  margin: 0 auto;
+  height: 100vh;
+}
+
+header{
+  background-color: black;
+  height: 110px;
+}
+
+
+h1 {
+  font-family: 'Merriweather', serif;  
+  color: yellow;
+  padding-left: 25px;
+  padding-top: 20px;
+  margin: 0;
+}
+
+nav {
+  display: inline;
+  float: right;
+  font-family: 'Source Sans Pro', sans-serif;
+  padding-right: 20px;
+  padding-top: 10px;
+}
+
+a.bold {
+  text-transform: uppercase;
+  font-size: 15px;
+  font-style: bold;
+  font-family: 'Oswald', sans-serif;
+
+}
+
+a:link {
+  color: white;
+  padding: 5px;
+  text-decoration: none;
+}
+
+hr {
+  margin: 0px;
+  background-color: yellow;
+  height: 3px;
+  border: none;
+}
+
+article {
+  background-image: url(../images/coworking.jpg);
+  background-size: 100%;
+  height: 475px;
+  background-color: red;
+}
+
+h2  {
+  font-family: 'Merriweather', serif; 
+  background-color: #C8C8C8;
+  font-size: 20px;
+  font-style: italic;
+  color: black;
+  text-align: center;
+  margin: 0;
+  padding: 17px;
+}
+
+h3 {
+  font-family: 'Merriweather', serif;
+}
+
+.remove {
+  color: #A0A0A0;
+  text-decoration: line-through;
+}
+
+.emphasize {
+  border-bottom: solid 2px yellow;
+  padding-bottom: .5px;
+  text-decoration-color: yellow
+}
+
+.transparentbox {
+  width: 264px;
+  background-color: rgba(225, 225, 225, 0.75);
+  padding: 15px;
+  padding-left: 20px;
+  position: relative;
+  left: 525px;
+  top: 25px;
+  box-shadow: 0px 0px 3px 1px #888888;
+  height: 370px;
+  font-family: 'Source Sans Pro', sans-serif;
+}
+
+.transparentbox h3 {
+  font-family: 'Source Sans Pro', sans-serif;
+  font-weight: bold;
+  font-size: 25px;
+}
+
+.transparentbox p {
+  font-size: 18px;
+  line-height: 26px;
+  padding-right: 5px;
+}
+
+aside.left, aside.center, aside.right {
+  background-color: white;
+  color: black;
+  margin: 0px;
+  padding-left: 20px;
```

Comment:
@jiniekwak this could have been written with the padding shorthand `padding: 25px 20px;`. Where the first value is top and bottom and the second value is left and right.